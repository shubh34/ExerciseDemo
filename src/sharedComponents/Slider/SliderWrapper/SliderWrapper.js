import React from 'react';
import PropTypes from 'prop-types';
import './SliderWrapper.scss';

const SliderWrapper = ({ children }) => (
	<div className="slider-wrapper">
		{children}
	</div>
);

SliderWrapper.propTypes = {
	children: PropTypes.arrayOf(PropTypes.element).isRequired,
};

export default SliderWrapper;
