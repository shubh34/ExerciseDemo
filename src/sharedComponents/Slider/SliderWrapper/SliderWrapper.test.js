
import React from 'react';
import { shallow } from 'enzyme';

import SliderWrapper from './SliderWrapper';


const setUp = (container = shallow) => {
	const wrapper = container(
		<SliderWrapper>
			<div>Wrapper</div>
		</SliderWrapper>,
	);
	return { wrapper };
};

describe('SliderWrapper', () => {
	it('should render childeren in wrapper', () => {
		const { wrapper } = setUp();
		expect(wrapper).toMatchSnapshot();
	});
});
