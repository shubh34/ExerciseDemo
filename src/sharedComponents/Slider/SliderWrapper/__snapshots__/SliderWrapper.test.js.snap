// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SliderWrapper should render childeren in wrapper 1`] = `
<div
  className="slider-wrapper"
>
  <div>
    Wrapper
  </div>
</div>
`;
